infrastructure/queues.tf

```diff
@@ -0,0 +1,4 @@
+module "sqs-splunk-queue" {
+  source = "./modules/sqs"
+  name   = "${terraform.workspace}-sqs-splunk-queue"
```

Comment:
Just for consistency, can you move the {terraform.workspace} prefix inside the module?